.travis.yml

```diff
@@ -52,23 +52,22 @@ jobs:
     # This tests the formatting of a PR.
     - if: (type = pull_request) OR (type = cron) OR NOT (branch = master)
       stage: test
-      name: "pgindent and license check"
+      name: "clang-format and license check"
       env:
-        - PG_VERSION=10.2 PG_GIT_TAG=REL_10_2 ADDITIONAL_CMAKE_FLAGS="-DUSE_DEFAULT_VISIBILITY=1"
+        - PG_VERSION=11.1 ASAN_OPTIONS="detect_leaks=0"
       before_install:
-        - git clone --branch ${PG_GIT_TAG} --depth 1 https://github.com/postgres/postgres.git /tmp/postgres
-        - docker run -d --name pgbuild -v ${TRAVIS_BUILD_DIR}:/build -v /tmp/postgres:/postgres timescaledev/pgindent:10-alpine
+        - docker run -d --name pgbuild -v ${TRAVIS_BUILD_DIR}:/timescaledb timescaledev/postgres-dev-clang:clang7-pg${PG_VERSION} /bin/sleep infinity
+      install:
+        - docker exec pgbuild /bin/bash -c "mkdir /tsdb_build && chown postgres /tsdb_build"
+        - docker exec pgbuild /bin/bash -c "cd /tsdb_build && cmake /timescaledb -DCMAKE_BUILD_TYPE=Debug -DPG_SOURCE_DIR=/usr/src/postgresql/ -DUSE_DEFAULT_VISIBILITY=1 && make"
       after_failure:
-        - docker exec -it pgbuild /bin/bash -c 'diff -r /build/src /tmp/timescale_src'
-        - docker exec -it pgbuild /bin/bash -c 'diff -r /build/test/src /tmp/timescale_test_src'
+        - git diff
       after_success:
       script:
-        - docker exec -it pgbuild /bin/bash -c 'cp -R /build/src/ /tmp/timescale_src && cp -R /build/test/src/ /tmp/timescale_test_src'
-        - docker exec -it pgbuild /bin/bash -c 'cd /build/debug && make pgindent'
-        - docker exec -it pgbuild /bin/bash -c 'diff -r -q /build/src /tmp/timescale_src'
-        - docker exec -it pgbuild /bin/bash -c 'diff -r -q /build/test/src /tmp/timescale_test_src'
-        - docker exec -it pgbuild /bin/bash -c 'cd /build/debug && make licensecheck'
-        - "test -z \"`./build/scripts/export_prefix_check.sh`\" || (echo \"functions missing 'ts_' prefix or should be static\"; ./build/scripts/export_prefix_check.sh; exit 1)"
+        - docker run -it -v ${TRAVIS_BUILD_DIR}:/timescaledb timescaledev/postgres-dev-clang:clang7-pg${PG_VERSION} /timescaledb/scripts/check_license_all.sh
+        - docker run -it -v ${TRAVIS_BUILD_DIR}:/timescaledb timescaledev/postgres-dev-clang:clang7-pg${PG_VERSION} /timescaledb/scripts/clang_format_all.sh
```

Comment:
Should this command, and the one above it use the `make` versions of these commands now that we're building timescale anyway? That way seems like it'll cause less breakage later on as we edit the scripts.